src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -552,7 +551,7 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
     if (!cy) {
       return;
     }
-    const duration = this.props.graphDuration.value;
+    const duration = store.getState().userSettings.duration;
```

Comment:
Minor, but I'm wondering why the store is being referenced directly.
The `CytoscapeGraph` component is wrapped and "connected" to redux at the bottom of this file. So, we should use `mapStateToProps` to bind to the state and use `this.props`. Same case with `store.dispatch`.